### PR TITLE
Update libnl3.rb

### PR DESCRIPTION
**Pull request by lovedheart**
#1981

### CODE REVIEW

File: packages/libnl3.rb [hunk 1]
```diff
@@ -3,32 +3,20 @@
...
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libnl3-3.2.25-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-    aarch64: '1eeb2dc6b831749cfdb466fe7daafed0bd4621b799782237ad972a2f46bf6566',
-     armv7l: '1eeb2dc6b831749cfdb466fe7daafed0bd4621b799782237ad972a2f46bf6566',
-       i686: '466eadbbc822bc03308d5f7881510de0cc61e5bfa7647b9600898018168a6d44',
-     x86_64: 'f29506472cccad26547d20afa519ed4823e2898a1822b43e7bc4a41f581d78c7',
-  })
-
   depends_on "buildessential" => :build
```
Comment (uberhacker) [inline packages/libnl3.rb]: This dependency can be removed since it's now part of Chromebrew "core".

File: packages/libnl3.rb [hunk 2]
```diff
@@ -3,32 +3,20 @@
...
-    system "./configure",
-      "--prefix=/usr/local",
-      "--sysconfdir=/usr/local/etc",
-      "--sbindir=/usr/local/sbin",
-      "--disable-static"
+	system "./configure \
+			    --prefix=#{CREW_PREFIX} \
+			    --libdir=#{CREW_LIB_PREFIX} \
+          --sysconfdir=#{CREW_DEST_PREFIX}/etc \
+          --sbindir=#{CREW_DEST_PREFIX}/sbin \
```
Comment (uberhacker) [inline packages/libnl3.rb]: These should be `#{CREW_PREFIX}` instead of `#{CREW_DEST_PREFIX}`.

File: packages/libnl3.rb [hunk 3]
```diff
@@ -3,32 +3,19 @@
...
-  })
-  binary_sha256 ({
-    aarch64: '1eeb2dc6b831749cfdb466fe7daafed0bd4621b799782237ad972a2f46bf6566',
-     armv7l: '1eeb2dc6b831749cfdb466fe7daafed0bd4621b799782237ad972a2f46bf6566',
-       i686: '466eadbbc822bc03308d5f7881510de0cc61e5bfa7647b9600898018168a6d44',
-     x86_64: 'f29506472cccad26547d20afa519ed4823e2898a1822b43e7bc4a41f581d78c7',
-  })
-
-  depends_on "buildessential" => :build
   depends_on "glibc"
```
Comment (uberhacker) [inline packages/libnl3.rb]: This is part of Chromebrew "core" also and can be removed.

Comment (lovedheart) [reply packages/libnl3.rb]: glibc also in core?

Comment (uberhacker) [reply packages/libnl3.rb]: Yes, see https://github.com/skycocker/chromebrew/pull/1949/files.

**Pull request merged**